NExtends/Primitives/Strings/String.extensions.cs

```diff
@@ -0,0 +1,564 @@
+
+                json = json.Replace("\\", "\\\\");
+
+                json = json.Replace("'", "\\'");
+                json = json.Replace("\"", "\\\"");
+
+                return json;
+            }
+            else
+            {
+                return "";
+            }
+        }
+
+        public static String JSONQuotesProtect(this String value)
+        {
+            if (value != null)
+            {
+                String json = value.Replace("\r\n", "<br>");
+                json = json.Replace("\n", "<br>");
+                json = json.Replace("\r", "<br>");
+
+                json = json.Replace("\\", "\\\\");
+
+                json = json.Replace("\"", "\\\"");
+
+                return json;
+            }
+            else
+            {
+                return "";
+            }
+        }
+
+        //Quelqueq méthodes pour convertir des types basiques en JSON
+        public static String ToJSON(this bool b)
+        {
+            return b.ToString().ToLower();
+        }
+        public static String ToJSON(this String s)
+        {
+            return JsonConvert.SerializeObject(s);//  "\"" + s.JSONQuotesProtect() + "\"";
+        }
+        public static String ToDoubleQuotesJSON(this String s)
+        {
+            return JsonConvert.SerializeObject(s);//return "\"" + s.JSONQuotesProtect() + "\"";
+        }
+        public static String ToDoubleQuotesJSON(this int i)
+        {
+            return i.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this double d)
+        {
+            return d.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this DateTime d)
+        {
+            return d.ToJQuery();
+        }
+        public static String ToDoubleQuotesJSON(this bool b)
+        {
+            return b.ToJSON();
+        }
+        public static String ToDoubleQuotesJSON(this object o)
+        {
+            if (o != null)
+            {
+                if (o.GetType() == typeof(int))
+                {
+                    return ((int)o).ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(double))
+                {
+                    return ((double)o).ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(DateTime))
+                {
+                    return ((DateTime)o).ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(Uri))
+                {
+                    return ((Uri)o).ToString().ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(string))
+                {
+                    return o.ToString().ToDoubleQuotesJSON();
+                }
+                else if (o.GetType() == typeof(bool))
+                {
+                    return ((bool)o).ToDoubleQuotesJSON();
+                }
+                //else if (o.GetType() == typeof(SearchResultEntry))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS125)